Configuration/PyReleaseValidation/python/relval_upgrade.py

```diff
@@ -13,21 +13,32 @@
 
 #just define all of them
 
+# special offsets
+trackingOnlyOffset = 0.1
+timingOffset = 0.2
+
 for year in upgradeKeys:
     for i,key in enumerate(upgradeKeys[year]):
         numWF=numWFAll[year][i]
         for frag in upgradeFragments:
             k=frag[:-4]+'_'+key
             stepList=[]
+            stepListTiming=[]
```

Comment:
Hi @kpedro88 - i've finally understood things - instead of these ifs, the correct change is to actually define correctly (in upgradeWorkflowComponents.py)

upgradeProperties[2023]['2023D4Timing'] 
(and other timing entries in the dictionary)

instead of copying it from something else. (of course the copy can be a good starting point if its just a matter of adding "Timing" to each object in the ScenToRun vector.

sorry for the delay, maybe I can even manage to propose a change, but not clear.